src/ParagraphNodeSpec.js

```diff
@@ -74,26 +72,44 @@ function getAttrs(dom: HTMLElement): Object {
   const styleLevel = parseInt(dom.getAttribute(ATTRIBUTE_STYLE_LEVEL), 10);
   // TODO: customStyle
 
-  return { align, indent, lineSpacing, paddingTop, paddingBottom, id, styleName, spacingAfterParagraph, styleLevel };
-}
-
-function toDOM(node: Node): Array<any> {
-  const {
+  return {
     align,
     indent,
     lineSpacing,
     paddingTop,
     paddingBottom,
     id,
     styleName,
+    spacingAfterParagraph,
```

Comment:
There is no difference , it was incorrectly named and removed these code since we picked these values from the style object which is returned from the server.